python/drned_xmnr/op/parse_log_errors.py

```diff
@@ -0,0 +1,71 @@
+def gather_problems(file):
+    problems = []
+    pending_pattern = None
+
+    matchless_max = None
+    matchless_to_go = None
+
+    for line_num, line in enumerate(file):
+        # TODO - beware - matcher with "lines-after" may interrupt
+        # other matcher if followed immediately in the log
+        matcher = _get_matcher(line)
+        if not matcher:
+            #  parse prefix-less lines for previously matched problem
+            if matchless_max and matchless_to_go:
```

Comment:
These two if-statements: are you checking for general false-like values, or for `None` only? Can you make that more obvious?

Reply:
updated...